camayoc/tests/remote/rho/test_scan.py

```diff
@@ -285,7 +203,10 @@ def scan_permutations():
         config = get_config()
         auths = [auth for auth in config['credentials']
                  if auth['type'] == 'network' and auth.get('rho', True)]
-        inventory = config['inventory']
+        inventory = {
+                machine['hostname']: machine for machine in config['inventory']
+                if machine.get('hypervisor') == 'vcenter'
+                }
```

Comment:
Forgot to update this earlier! Now only test cases will be generated for vcenter machines -- which makes sense to me at the moment. This could be changed to something else if we end up with another hypervisor for network hosts.